packages/next-ui/Layout/components/LayoutHeaderBack.tsx

```diff
@@ -12,7 +12,9 @@ import { IconSvg } from '../../IconSvg'
 import { responsiveVal } from '../../Styles'
 import { iconChevronLeft } from '../../icons'
 
-export type BackProps = Omit<LinkOrButtonProps, 'onClick' | 'children'>
+export type BackProps = Omit<LinkOrButtonProps, 'onClick' | 'children'> & {
+  preventClickBack?: boolean
```

Comment:
From the code this doesn't really become clear what this does. At least it should have a comment what this does.


```
/**
 * Will not use `router.back()` if available, and will always use the `up.href`
 */
disableBackNavigation?: boolean
```